src/components/Image/Circle.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import colors from "../../theme/colors";
+
+const Image = styled.img`
+  border: solid 1px ${colors.lightGray};
+  align-items: center;
+  justify-content: center;
+  width: ${props => props.size}px;
+  height: ${props => props.size}px;
+  border-radius: 50%;
+`;
+
+const CircleImage = ({ size, src, alt, ...props }) => (
```

Comment:
- ThumbnailCircle? 
- also consider taking an `image` render prop so we can provide a lazy loading component instead of just `src` which will load the image when we don't need it. `src` can be taken in as a fallback.